packages/server/postgres/queries/src/backfillSegmentIdQuery.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  @name backfillSegmentIdQuery
+  @param users -> ((segmentId, id)...)
+*/
+UPDATE "User" AS u SET
+  "segmentId" = c."segmentId"
+FROM (VALUES :users) AS c("segmentId", "id") 
+WHERE c."id" = u."id";
```

Comment:
looks good!

one nitpick i have is that for one-off queries like this, the need for a defined type-safe api in the app isn't as great, so going through the whole pgtyped build process may be overkill. it might make more sense to use the passed `pgm` client or `getPg` here. pgm uses the node-postgres driver under the hood, so either one should be equivalent.

Reply:
I like simpler code but I'm not sure a good way to do [argument expansion](https://pgtyped.vercel.app/docs/sql-file#array-spread-and-pick) without pgtyped? This is sth I tried:

```typescript
      const users_expanded = pgUsers.map(({segmentId, id}) => `('${segmentId}', '${id}')`).join(',')
      pgm.sql(`
        UPDATE "User" AS u SET
        "segmentId" = c."segmentId"
        FROM (VALUES ${users_expanded}) AS c("segmentId", "id") 
        WHERE c."id" = u."id";
      `)
```

Reply:
nice job! that's what i would have done. i prefer the version without pgtyped, but don't have too much tension here.